xclim/sdba/nbutils.py

```diff
@@ -141,7 +283,15 @@ def remove_NaNs(x):  # noqa
     return x[:, ~remove]
 
 
-@njit(fastmath=True)
+@njit(
+    [
+        float32(float32[:, :], float32[:, :]),
+        float64(float64[:, :], float64[:, :]),
+    ],
+    fastmath=True,
```

Comment:
FYI, this is outside the scope of this PR, but there might be a bug here with fastmath=True, when there are NaNs in the series.

Reply:
https://llvm.org/docs/LangRef.html#fast-math-flags

fastmath=True implies nnan and ninf, both of which are not always true for our data. Is there a check in the function that calls this?

Reply:
For those functions, there are no Nans involved, this is after calling `remove_NaNs` in `_escore` so all's good!